### PR TITLE
CONTRIBUTING: Update the release schedule: Increase cadence by a week

### PR DESCRIPTION
**Pull request by eskultety**
2 weeks may be too slow for some ultra-agile, rapid development consumers of this project, often having led to breaking the scheduled pattern and resorting to releases on an ad-hoc basis. Let's shorten the schedule by a week, describing scenarios at which a given release may be at risk.

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] Docs updated (if applicable)
- [ ] Docs links in the code are still valid (if docs were updated)

**Note:** if the contribution is external (not from an organization member), the CI
pipeline will not run automatically. After verifying that the CI is safe to run:

- [approve GitHub Actions workflows][approve-gh-actions] by clicking a button
- approve the Red Hat Trusted App Pipeline container build by commenting `/ok-to-test`
  (as is the [standard for Pipelines as Code][pac-running-pipeline])

[approve-gh-actions]: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
[pac-running-pipeline]: https://pipelinesascode.com/docs/guide/running/#running-the-pipelinerun

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -289,5 +289,8 @@ upon publishment.*
 
 ## Release schedule
 
-This project follows a two-week release cycle. Note that a release might be skipped if there's no
-significant changes to release.
+This project follows a weekly release schedule, with planned releases every Tuesday. If there is
+no significant content to ship on a given release day, we automatically defer to the next scheduled
```
Comment (a-ovchinnikov) [inline CONTRIBUTING.md:293]: s/we automatically defer to the next scheduled window/we may skip a release/

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -289,5 +289,8 @@ upon publishment.*
...
+window. Should we encounter any CI issues on the day of a release, we postpone until after those
```
Comment (a-ovchinnikov) [inline CONTRIBUTING.md:294]: s/, we postpone until.*\./, we either release immediately after the issue is resolved (if a release is urgent) or skip the release (if no urgent changes were planned)./

**Review by a-ovchinnikov — APPROVED** [review]
LGTM with some optional minor suggestions.

**Review by slimreaper35 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (eskultety) [reply CONTRIBUTING.md:294]: How do you define an "urgent release" when it comes to regular releases and more specifically regarding features/improvements, i.e. not bug/security fixes? I mean I consider any feature/improvement (apart from experimental stuff naturally) equally important when it comes to releasing it, no skipping lines.

How about:
_we either release immediately after the issue is resolved or skip the release based on how long it takes to get the CI green again_ ?

Comment (a-ovchinnikov) [reply CONTRIBUTING.md:294]: "Urgent" is a blocking or security bug, everything else is regular. Urgent can coincide with regular release. "Skip ... based on how long it takes to..." shifts focus from release nature to CI block nature which I think is not right. I would have rather kept the original wording than update it this way: I tried to make the sentence a bit easier to digest, but it is fine as it is now.

Comment (eskultety) [reply CONTRIBUTING.md:294]: > Urgent can coincide with regular release.

True. I will update with your suggestion, but will swap the last 2 sentences as it makes more sense that way with the suggestion.

**Review by a-ovchinnikov — APPROVED** [review]

**Pull request merged**